contrib/test_jenkins.sh

```diff
@@ -239,19 +239,28 @@ run_hello() {
 
 	sleep 5
 
+	# temporary disable 
+	if [[ ${test_args} == *"-e"* ]]
+	then
+		set +Ee
+	fi
+
 	# need to be ran in background to reflect application PID in $!
 	./${test_name} ${test_args} -n $(hostname) -p ${tcp_port} &
 	hw_client_pid=$!
 
 	# make sure server process is not running
 	wait ${hw_server_pid} ${hw_client_pid}
+
+	# return default
+	set -Ee
```

Comment:
Does `set` command affect all sub-shells? Server starts before `set +Ee`, so i thought it shouldn't be affected.